proposed-level-2-roadmap.md

```diff
@@ -52,6 +52,11 @@ remains a fairly blunt opt-in mechanism that developers have to know about and c
 Priming](https://wicg.github.io/hsts-priming/) aimed to bump the defaults up a bit, but hasn't shown
 enough value in based on the rumblings I've heard from Mozilla to be worth implementing.
 
+### Downloads
+
+Insecure downloads can be highly dangerous, but they are not treated as mixed content, and many
+browsers (including Chrome) do not even label them as insecure in the UI.
```

Comment:
Is it worth additionally mentioning that Chrome doesn't (and _can't_) downgrade security indicators when insecure downloads may be present?